examples/sbol2/CreatingSBOL2Objects/Component.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating a Component\n",
+    "\n",
+    "`Components` connect component `ComponentDefinitions` into a hierarchy. In this totorial, we will have one a parent `ComponentDefinition` representing a gene and a child `ComponentDefinition` representing a promoter on said gene.\n",
```

Comment:
```suggestion
    "`Components` connect `ComponentDefinitions` into a hierarchy. In this tutorial, we will have one a parent `ComponentDefinition` representing a gene and a child `ComponentDefinition` representing a promoter on said gene.\n",
```

I also think that the word "gene" is problematically ambiguous. I would suggest instead talking about this as an "expression construct" that has both a promoter and a CDS, and adding both of them in with a constraint between them.